### ISSUE TITLE
Let's collaborate!

### ISSUE THREAD

**Issue by ndelangen** [chinchiheather/storybook-addon-prettier-source#2]
Hi @chinchiheather,

This looks very good!, Can I ask for your help with the upcoming docs-view epic for storybook? https://storybook.canny.io/roadmap/p/docs-view-mode

Looks like you have an interest in doing documentation on top of storybook. 

Right now I'm working on a big overhaul of the UI, your opinion on that is welcome also!
https://github.com/storybooks/storybook/pull/4086

**Comment by chinchiheather** [comment]
Hi @ndelangen 

Thanks! Sure, I can help, but I am not sure exactly what sort of help you mean? Help working on features?

**Comment by ndelangen** [comment]
Looking for people who want to help with anything really.

Feedback is easiest possibly, but discussing what the features should look like is also great.

If we can use some of your code or knowledge building docs-view, that'd be great too.

In storybooks/storybook#4086 I made a SyntaxHighlighter component, but it currently doesn't run the code though prettier right now.

I was thinking of either running prettier in babel so there's no runtime overhead.

Or alternatively run prettier in the browser and actually use the width of the div to make prettier wrap lines.

I found your repo trying to find a Component that does the above btw 😄

**Comment by chinchiheather** [comment]
Right, yeah I found it pretty easy to use the browser version of prettier, but I'm not sure exactly what the overhead of it is. You can still use all the prettier options in the browser one, so I guess you could calculate the width of the div and set that as the `printWidth` if that's the route you ended up going.

I'm finding it kind of hard to visualise what the docs-view is exactly. Is it a different thing to the UI overhaul or part of it?

I'm happy to help, but maybe some concrete tasks would be good? 😉 Like, is there a specific place to be leaving feedback? Or are there features I can work on?

**Comment by ndelangen** [comment]
Adding prettier on the SyntaxHighlighter component would be really cool starting point I think. That would be part of the UI overhaul.

The docs view epic will follow after.

Maybe we should get a skype/hangout meeting scheduled?

**Comment by chinchiheather** [comment]
Okay, sounds good!

Yeah we can arrange a meeting if you like, I live in Australia btw

**Comment by ndelangen** [comment]
Cool, can you meet me on discord https://discord.gg/sMFvFsG or slack https://now-examples-slackin-rrirkqohko.now.sh/

I'm flexible on time 👍
